packages/edit-site/src/components/sidebar-navigation-screen-page/page-details.js

```diff
@@ -112,10 +117,10 @@ export default function PageDetails( { id } ) {
 					? postContext?.templateSlug
 					: null;
 			const _templateTitle =
-				defaultTemplateTypes && templateSlug
-					? defaultTemplateTypes.find(
+				templates && templateSlug
+					? templates?.find(
```

Comment:
is the optional chaining required here or would templates be something other than array if it exists?

Reply:
No, it's a leftover from something I had before 😄